packages/dbml-parse/src/lib/analyzer/validator/elementValidators/tableGroup.ts

```diff
@@ -70,11 +70,31 @@ export default class TableGroupValidator implements ElementValidator {
   }
 
   private validateSettingList(settingList?: ListExpressionNode): CompileError[] {
-    if (settingList) {
-      return [new CompileError(CompileErrorCode.UNEXPECTED_SETTINGS, 'A TableGroup shouldn\'t have a setting list', settingList)]
+    const aggReport = aggregateSettingList(settingList);
+    const errors = aggReport.getErrors();
+    const settingMap = aggReport.getValue();
+
+    for (const name in settingMap) {
+      const attrs = settingMap[name];
+      switch (name) {
+        case 'headercolor':
+          errors.push(...attrs.map((attr) => new CompileError(CompileErrorCode.INVALID_TABLE_SETTING, '\'headercolor\' is not supported', attr)))
```

Comment:
You are missing a lot of `;` at the end of a line. Please use eslint and fix the code style in others places in your code as well.